src/translations/en/admin.json

```diff
@@ -1,3 +1,14 @@
 {
-  "add_institution": "Add institution"
+  "add_institution": "Add institution",
+  "administrationId": "Authenticating id for institution administrator",
+  "archiveName": "Archive name",
+  "cname": "CNAME",
+  "displayName": "Display name for the institution",
+  "feedback": {
+    "required_field": "Required field"
+  },
```

Comment:
har vi ikke denne en annen plass?

Reply:
joda- for publikasjon, men da var den en del av flere feedback-meldinger. Kanskje bytter vi ut teksten med mer spesifike tekster - som "Navn er påkrevd". Pr nå vet jeg ikke hva som er påkrevd engang